### PR TITLE
envoy: optimise fuzzers

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Disables instrumentation in various libraries to increase fuzzing performance. @asraa

### CODE REVIEW

File: projects/envoy/build.sh [hunk 1]
```diff
@@ -38,9 +38,36 @@ fi
...
+  echo " --per_file_copt=^.*com_googlesource_code_re2.*\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*upb.*\.cpp\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*org_brotli.*\.cpp\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*com_google_cel_cpp.*\.cpp\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*com_github_jbeder_yaml_cpp.*\.cpp\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*proxy_wasm_cpp_host/.*\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*com_github_google_libprotobuf_mutator/.*\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*com_googlesource_googleurl/.*\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*com_lightstep_tracer_cpp/.*\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
+  echo " --per_file_copt=^.*\.pb\.cc\$@-fsanitize-coverage=0,-fno-sanitize=all"
```
Comment (asraa) [inline projects/envoy/build.sh:62]: Could you add a comment about protocol buffers as well?

**Review by asraa — APPROVED** [review]
Thanks! Small nit, not blocking.

File: projects/envoy/build.sh (hunk 1 above)
Comment (DavidKorczynski) [reply projects/envoy/build.sh:80]: Done!

**Comment by DavidKorczynski** [comment]
FYI CI fails due to long build time of Envoy.

**Comment by asraa** [comment]
> FYI CI fails due to long build time of Envoy.

Yep :/ Happens. Assuming a local test with docker image went ok?

**Review by inferno-chromium — APPROVED** [review]
yes, sometime we can think how ci can be fixed for envoy, not urgent since prod should be fine.

**Pull request merged**